### ISSUE TITLE
Search with pagination

### ISSUE THREAD

**Issue by julienwangchang** [bjorn2404/jQuery-Store-Locator-Plugin#79]
Hi,

When I am on page 2 and search again we stay on page 2 instead of coming back on page 1.
Is it possible to come back page 1 when clicking on search button while on other page ?

Thanks

**Comment by bjorn2404** [comment]
Hello,

Hmm.. yeah it should reset. I'll get that fixed in the next couple of days.

Thanks

**Comment by bjorn2404** [comment]
Ok, I've pushed a fix to the dev branch but I've got a couple of other things I want to do before pushing to master. If you are looking for a fix in the meantime you can update the page check that starts on line  1629 to:

```
// Set the initial page to zero if not set
if ( _this.settings.pagination === true ) {
    if (typeof page === 'undefined' || originalOrigin !== addressInput ) {
        page = 0;
    }
}
```

**Comment by bjorn2404** [comment]
Fixed in v2.2.0
